ocs_ci/deployment/fusion_data_foundation.py

```diff
@@ -137,3 +142,45 @@ def fusion_service_instance_health_check():
     install_percent = instance_status["installStatus"]["progressPercentage"]
     assert service_health == "Healthy"
     assert install_percent == 100
+
+
+def setup_fdf_pre_release_deployment():
+    """
+    Perform steps to prepare for a Pre-release deployment of FDF.
+    """
+    fdf_image_tag = config.DEPLOYMENT.get("fdf_image_tag")
+    fdf_catalog_name = defaults.FUSION_CATALOG_NAME
+    fdf_registry = config.DEPLOYMENT.get("fdf_pre_release_registry")
```

Comment:
where do we get/pass "fdf_pre_release_registry" to ocs-ci. didn't see this any yaml files.

Reply:
Should this be:
fdf_pre_ga_registry?

Reply:
This got mixed up in my commit somehow but I've refactored and have started using `pre_release` instead of `pre_ga` in all instances.